Description: use x-terminal-emulator instead of xterm
 x-terminal-emulator is a sensible default terminal emulator on Debian, so
 use it instead of xterm in the default configuration.
Forwarded: not-needed
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2009-09-23
Index: scrotwm.git/scrotwm.conf
===================================================================
--- scrotwm.git.orig/scrotwm.conf	2011-07-04 12:53:06.756084343 +0200
+++ scrotwm.git/scrotwm.conf	2011-07-04 13:01:34.698881517 +0200
@@ -25,13 +25,13 @@
 #border_width			= 1
 
 # spawn app
-# program[term]		= xterm
+# program[term]		= x-terminal-emulator
 # program[screenshot_all]	= screenshot.sh full
 # program[screenshot_wind]	= screenshot.sh window
 # program[lock]		= xlock
 # program[initscr]	= initscreen.sh
 # program[menu]		= dmenu_run -fn $bar_font -nb $bar_color -nf $bar_font_color -sb $bar_border -sf $bar_color
-spawn_term		= xterm
+spawn_term		= x-terminal-emulator
 
 # dialog box size ratio .3 >= r < 1
 dialog_ratio		= 0.6
Index: scrotwm.git/scrotwm.c
===================================================================
--- scrotwm.git.orig/scrotwm.c	2011-07-04 13:00:54.514373013 +0200
+++ scrotwm.git/scrotwm.c	2011-07-04 13:01:34.702881568 +0200
@@ -3881,7 +3881,7 @@
 void
 setup_spawn(void)
 {
-	setconfspawn("term",		"xterm",		0);
+	setconfspawn("term",		"x-terminal-emulator",		0);
 	setconfspawn("screenshot_all",	"screenshot.sh full",	0);
 	setconfspawn("screenshot_wind",	"screenshot.sh window",	0);
 	setconfspawn("lock",		"xlock",		0);
@@ -5906,7 +5906,7 @@
 	if ((bar_fonts[2] = strdup("-misc-fixed-medium-r-*-*-*-*-*-*-*-*-*-*"))
 	    == NULL)
 		err(1, "setup_globals: strdup");
-	if ((spawn_term[0] = strdup("xterm")) == NULL)
+	if ((spawn_term[0] = strdup("x-terminal-emulator")) == NULL)
 		err(1, "setup_globals: strdup");
 	if ((clock_format = strdup("%a %b %d %R %Z %Y")) == NULL)
 		errx(1, "strdup");
